devices.js

```diff
@@ -1868,6 +1868,25 @@ const devices = [
             execute(device, actions, callback);
         },
     },
+    {
+        zigbeeModel: ['outlet'],
+        model: 'IM6001-OTP05',
+        vendor: 'SmartThings',
+        description: 'Outlet',
+        supports: 'on/off',
+        fromZigbee: [fz.state, fz.ignore_onoff_report],
+        toZigbee: [tz.samsung_on_off],
```

Comment:
Does it also work when using `tz.on_off` instead of `tz.samsung_on_off`?

Reply:
no, crashes :(

Reply:
sorry, wrong spot, doesn't crash... nothing happens.  the changes i made seem to work.

Reply:
I would like to get a better understanding why this doesn't work. Could you post the log with `advanced -> log_level: debug` with both converters when turning on/off the switch?

Reply:
you want me to change code to
`toZigbee: [tz.samsung_on_off, tz.on_off]]` 
and log... or just
`toZigbee: [tz.on_off]]`

Reply:
just `toZigbee: [tz.on_off]`

Reply:
strange... working now... i swear it didn't first time... perhaps a timing/reload issue (didn't save file properly before reloading).